paddle/fluid/framework/parallel_executor.cc

```diff
@@ -270,6 +290,12 @@ void ParallelExecutor::BCastParamsToDevices(
 
 void ParallelExecutor::Run(const std::vector<std::string> &fetch_tensors,
                            const std::string &fetched_var_name) {
+#ifdef WITH_GPERFTOOLS
+  if (gProfileStarted) {
+    ProfilerFlush();
```

Comment:
Using `ProfilerFlush` instead of `ProfilerStop` because we always kill our program instead of waiting it done. So we can kill our program and generate profile result by using `ProfilerFlush`,